models/theme.js

```diff
@@ -0,0 +1,36 @@
+var mongoose = require('mongoose');
+var Schema = mongoose.Schema;
+
+var connection = mongoose.createConnection("mongodb://localhost/niji");
```

Comment:
Is it normal to create a connection in each model? It seems like it would be better to use a singleton connection. Also, we should make this configurable for production.